src/redux/store.js

```diff
@@ -1,11 +1,15 @@
 import { configureStore } from '@reduxjs/toolkit'
-import itemReducer from './items/items'
+import itemsSlice from './items/items'
 import userReducer from './users/user'
 
 export const store = configureStore({
   reducer: {
-    items: itemReducer,
+    itemsSlice,
     user: userReducer
   },
-  devTools: true
+  devTools: true,
+  middleware: getDefaultMiddleware =>
```

Comment:
Only plain serializable objects, arrays, and primitives should be put in our store, so when we put in Items and Location which were mapping over  <options> element there was an error being thrown:
`react_devtools_backend.js:4026 A non-serializable value was detected in an action` 
Since this is an occasion when we have to deal with actions that need to accept non-serializable data, we can turn off the serializable check in middleware.
referenceLinks: [1](https://redux.js.org/faq/organizing-state#can-i-put-functions-promises-or-other-non-serializable-items-in-my-store-state), [2](https://redux-toolkit.js.org/usage/usage-guide#working-with-non-serializable-data)